tests/lca/imagebasedinstall/mgmt/deploy/tests/e2e-deploy-test.go

```diff
@@ -310,3 +116,281 @@ var _ = Describe(
 				"error: extra manifest configmap has incorrect content")
 		})
 	})
+
+func createSharedResources() {
+	By("Create namespace for IBI installation")
+
+	_, err := namespace.NewBuilder(APIClient, MGMTConfig.Cluster.Info.ClusterName).Create()
+	Expect(err).NotTo(HaveOccurred(), "error creating namespace")
+
+	By("Get pull secret from hub cluster")
+
+	spokePullSecret, err := secret.Pull(APIClient, "pull-secret", "openshift-config")
+	Expect(err).NotTo(HaveOccurred(), "error getting pull-secret from hub cluster")
+
+	By("Create pull secret for spoke cluster")
+
+	spokePullSecret.Definition.Name = MGMTConfig.Cluster.Info.ClusterName
+	spokePullSecret.Definition.Namespace = MGMTConfig.Cluster.Info.ClusterName
+	spokePullSecret.Definition.ResourceVersion = ""
+	_, err = spokePullSecret.Create()
+	Expect(err).NotTo(HaveOccurred(), "error creating spoke pull-secret")
+
+	if MGMTConfig.ExtraManifests {
+		By("Create namespace for extramanifests")
```

Comment:
```suggestion
		By("Create namespace builder to support creating a namespace with extramanifests")
```
Or similar to above, or get the impression that the extramanifests are in their own namespace